src/shortcuts.js

```diff
@@ -0,0 +1,18 @@
+const createShortcut = method => ({ options = {}, ...rest} = {}) => ({
+    ...rest,
+    options: {
+        ...options,
+        method
```

Comment:
What if I'll use shortcut and method in options? I see shortcut's the most priority argument, but are you sure it is the right way? I think case when you are passing method through arguments more usual case.